app/src/main/java/app/accrescent/client/ui/SettingsScreen.kt

```diff
@@ -203,6 +186,73 @@ fun Setting(
     }
 }
 
+/**
+ * Modifier to get rounded corners on the ripple that appears when applying
+ * a clickable/toggleable/etc to a composable.
+ *
+ * @param modifier clickable/toggleable/etc modifier with an indication.
+ * @return An interactable [Modifier] with a rounded indication.
+ */
+private fun Modifier.roundedInteractable(modifier: Modifier = Modifier): Modifier =
+    this
+        .clip(RoundedCornerShape(8.dp))
+        .then(modifier)
+        .padding(8.dp)
```

Comment:
I really like moving this into its own modifier (and the doc comments). However, this implementation seems unnecessarily complex and probably the wrong abstraction. There's nothing requiring the passed modifier to be "interactable," and passing a modifier as a parameter to a custom modifier is strange since you can just use the `this` in scope. For example,

```kotlin
private fun Modifier.rounded(): Modifier =
    clip(RoundedCornerSHape(8.dp)
        .then(this)
        .padding(8.dp)
}
```

appears to accomplish the same thing.